frontend/server/minio-helper.test.ts

```diff
@@ -19,7 +19,6 @@ import { createMinioClient, isTarball, maybeTarball, getObjectStream } from './m
 import { V1beta1JobTemplateSpec } from '@kubernetes/client-node';
 
 jest.mock('minio');
-jest.mock('./aws-helper');
```

Comment:
why is this removed? (I have no context on these tests, just curious)

Reply:
Was removed in the initial implementation since I had removed all the previous mocks that used aws-helper. Now that am reverting those changes back this will be necessary once again good catch.